### PR TITLE
Update manifest schema reference

### PR DESCRIPTION
**Pull request by JoshuaBehrens**
When you copy the reference you'll get lots of xsd related issues (depending on the IDE/editor). When we use the latest reference we are good to go, aren't we?

### CODE REVIEW

File: resources/references/app-reference/manifest-reference.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 {% code title="manifest.xml" %}
 ```markup
 <?xml version="1.0" encoding="UTF-8"?>
-<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/app-system/0.1.0/src/Core/Content/App/Manifest/Schema/manifest-1.0.xsd">
+<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/app-system/0.2.0/src/Core/Content/App/Manifest/Schema/manifest-1.0.xsd">
```
Comment (keulinho) [inline resources/references/app-reference/manifest-reference.md:6]: definetly an error that we still reference the app-system plugin, as the app-system now is integrated into the platform

so we should use the current schema file from the platform repo, please use the `https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd` url

```suggestion
<manifest xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://raw.githubusercontent.com/shopware/platform/trunk/src/Core/Framework/App/Manifest/Schema/manifest-1.0.xsd">
```

**Review by keulinho — CHANGES_REQUESTED** [review]
@JoshuaBehrens thanks for pointing out the outdated reference, but please update it to the current platform schema, as that link should not get outdated anytime sonn.

File: resources/references/app-reference/manifest-reference.md (hunk 1 above)
Comment (JoshuaBehrens) [reply resources/references/app-reference/manifest-reference.md:6]: Using trunk is a good one so you are not missing to update again :)

**Comment by JoshuaBehrens** [comment]
@keulinho do we need a squash for that?

**Review by keulinho — APPROVED** [review]

**Comment by keulinho** [comment]
I don't think a squash is needed here

**Pull request merged**